Tools/AnyMocap/DrawVector.any

```diff
@@ -9,7 +9,11 @@ Line = {
 };
 Opacity = iffun(bool(sum(...Driver.WeightPos)), 1.0, 0.2);
 #var AnyVar size=.Arrows.size;
+#ifdef _MOCAP_HIDE_MARKER_ARROWS_
+#var Visible = Off;
+#else
```

Comment:
I couldn't get this define statement to work. I would love to know how this works.

Reply:
Ok. I will take a closer look. I recall that there was some acutal bugs with model view here.